src/main/java/com/hm/achievement/utils/FileUpdater.java

```diff
@@ -345,6 +345,10 @@ private void updateSetting(CommentedYamlConfiguration file, String name, Object
 			updatePerformed = true;
 		}
 	}
+	
+	private void updateLang(CommentedYamlConfiguration file, Lang lang) {
```

Comment:
I think it would be as simple as the test that checks for lang.yml keys.

Good that you commented on this because I made that method so that the possibility to do so remains open.